features/aanspreekvorm.feature

```diff
@@ -0,0 +1,659 @@
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde          |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm> |
+
+      Voorbeelden:
+      | geslacht | naamgebruik | adellijke titel | titel/predicaat partner | aanspreekvorm            |
+      | M        | E           | R               |                         | De hoogwelgeboren heer   |
+      | V        | V           | G               |                         | De hooggeboren vrouwe    |
+      | O        | N           | P               |                         | De hoogheid              |
+      | V        | N           | BS              | P                       | De hoogwelgeboren vrouwe |
+      | M        | E           | G               | JV                      | De hooggeboren heer      |
+      | O        | V           | PS              | BS                      | De hoogheid              |
+
+    Abstract Scenario: persoon met geslacht "<geslacht>", adellijke titel en aanduiding naamgebruik "P", heeft partner met/zonder adellijke titel/predicaat
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde     |
+      | burgerservicenummer         | 999992934  |
+      | geslachtsaanduiding (04.10) | <geslacht> |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde |
+      | adellijke titel of predicaat (02.20) | G      |
+      | aanduiding naamgebruik (61.10)       | P      |
+      En de persoon heeft een 'partner' met de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam                                                               | waarde   |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | 20180808 |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                                 | waarde                    |
+      | adellijke titel of predicaat (02.20) | <titel/predicaat partner> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
+      Voorbeelden:
+      | geslacht | titel/predicaat partner |
+      | M        |                         |
+      | V        |                         |
+      | O        |                         |
+      | M        | JH                      |
+      | V        | GI                      |
+      | O        | P                       |
+
+  Rule: Er is geen aanspreekvorm als de persoon een predicaat heeft, het geslacht gelijk is aan "V" en een actuele partner heeft
+
```

Comment:
Is een scenario met de titel "persoon met predicaat, heeft geen actuele partner" hier niet ook gewenst?